.travis.yml

```diff
@@ -7,6 +7,12 @@ language: java
 cache:
   directories:
     - ~/.m2
+    - ~/.ivy2
+
+addons:
+  apt:
+    packages:
+      - ant
 
```

Comment:
We add and remove addons.
In some recent commit Nick suggested more stable way to install packages

Reply:
I am not familar with any new ways. I just copied this from samples repo to reproduce it's CI.

Reply:
Can you add ant to `sudo apt install -y .....`
That we already have for installation. Look at commit from my comment

Reply:
I think done.